skactiveml/pool/_qbc.py

```diff
@@ -61,32 +68,35 @@ class QBC(SingleAnnotPoolBasedQueryStrategy):
     [1] H.S. Seung, M. Opper, and H. Sompolinsky. Query by committee.
         In Proceedings of the ACM Workshop on Computational Learning Theory,
         pages 287-294, 1992.
-    [2] N. Abe and H. Mamitsuka. Query learning strategies using boosting and bagging.
-        In Proceedings of the International Conference on Machine Learning (ICML),
-        pages 1-9. Morgan Kaufmann, 1998.
+    [2] N. Abe and H. Mamitsuka. Query learning strategies using boosting and
+        bagging. In Proceedings of the International Conference on Machine
+        Learning (ICML), pages 1-9. Morgan Kaufmann, 1998.
     """
 
-    def __init__(self, clf, ensemble=None, method='KL_divergence', missing_label=MISSING_LABEL, random_state=None,
-                 **kwargs):
+    def __init__(self, clf, ensemble=None, method='KL_divergence',
+                 classes=None, missing_label=MISSING_LABEL, random_state=None,
+                 **kwargs_ensemble):
```

Comment:
please use a dictionary here not kwargs. call it `ensemble_dict=None`. check if dict in `query`

Reply:
__init__ cannot have kwargs